aiida/backends/tests/test_base_dataclasses.py

```diff
@@ -97,174 +103,188 @@ def test_store_load(self):
 
 
 class TestFloat(AiidaTestCase):
+    """Test Float class."""
 
     def setUp(self):
         super().setUp()
         self.value = Float()
         self.all_types = [Int, Float, Bool, Str]
 
     def test_create(self):
-        a = Float()
+        """Creating basic data objects."""
+        term_a = Float()
```

Comment:
to be honest, when variables stand for "arbitrary mathematical variables", one- or two-letter variables should be perfectly fine.
The reason the rule for >=3 characters exists is to convey the meaning of the variable.
In this context `a` conveys the meaning of the variable just fine.

Reply:
I hope this does not mean "replace everything back" :)